.travis.yml

```diff
@@ -34,6 +36,8 @@ before_script:
 script:
     - cd $TRAVIS_BUILD_DIR/svir && make test
     - cd $TRAVIS_BUILD_DIR/svir && make integrationtest
+    - cd $TRAVIS_BUILD_DIR/svir && make build_apidoc
```

Comment:
A minor comment. You may want to move the documentation generation in the `after_succes` block (see https://github.com/gem/oq-engine/blob/master/.travis.yml#L39)